pubsub/gochannel/pubsub_test.go

```diff
@@ -8,12 +8,14 @@ import (
 	"testing"
 	"time"
 
+	"github.com/ThreeDotsLabs/watermill/pubsub/gochannel"
+
+	"github.com/ThreeDotsLabs/watermill/pubsub/tests"
```

Comment:
This is just cosmetics, but could be sorted properly.

Reply:
done****